astronomer/providers/amazon/aws/hooks/s3.py

```diff
@@ -215,7 +215,7 @@ async def is_keys_unchanged(
         inactivity_seconds: int = 0,
         allow_delete: bool = True,
         last_activity_time: Optional[datetime] = None,
-    ) -> Dict[str, str]:
+    ) -> Dict[str, Any]:
```

Comment:
why changed str to Any?

Reply:
it returns  str objects and DateTime object too now in the dict

Reply:
Try Dict[str, Union[str, datetime, int]]

Reply:
Values for the dict is being parsed on Trigger and the value for the key can be anything rather than just one type.